scripts/aws/setupdb.sh

```diff
@@ -131,4 +131,7 @@ if [ "$load_water_quality" = "true" ] ; then
     FILES=("nhd_water_quality.sql.gz" "drb_catchment_water_quality.sql.gz")
     PATHS=("drb_catchment_water_quality_tn" "drb_catchment_water_quality_tp"
             "drb_catchment_water_quality_tss" "nhd_quality_tp" "nhd_quality_tn")
+
+    download_and_load $FILES
+    purge_tile_cache $PATHS
```

Comment:
The AWS CLI only gets installed when the production `requirements.txt` file is used. Might be able to do something like this to side-step the errors locally:

```bash
if which aws > /dev/null; then
    aws ...
fi
```